src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -100,18 +104,19 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         // No validation
-        internal CSharpParseOptions(
+        private CSharpParseOptions(
```

Comment:
I opened bug to clean up ctor issues - https://github.com/dotnet/roslyn/issues/16475